template/src/Layout/__tests__/Layout.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { Layout } from '../Layout';
+
+describe('Layout', () => {
+  it('should display the header, main and footer', () => {
+    render(
+      <Layout>
+        <div>Contenu test</div>
```

Comment:
c'est juste un test je sais mais cela n'exclut pas d'utiliser une bonne sémantique 😆 
```suggestion
        <p>Contenu test</p>
```

Reply:
done